### ISSUE TITLE
Allow two-page layout view option in Universal Viewer

### ISSUE THREAD

**Issue by nikdragovic** [emory-libraries/dlp-lux#286]
## User Story
As an end user, I want to see a side-by-side view of a text object such as a book in the Universal Viewer, so that I can view it in a context that approximates the physical object.

Recreating #276 due to apparent Zenhub issue.

## Acceptance Criteria
- [x] User can click a button in the UV to view object component images side-by-side
- [x] For a digitized book, users should be able to see facing pages together (as opposed to pages that are on the same leaf). Optional, will investigate whether this is possible.

Implementation note: 
A good source of documentation for UV configuration is https://github.com/UniversalViewer/universalviewer/blob/master/src/extensions/uv-seadragon-extension/config/en-GB.json. 
Example images below.

**Comment by nikdragovic** [comment]
Example object from Curate: https://curate-arch.library.emory.edu/concern/curate_generic_works/000000002j-cor?locale=en

**Comment by nikdragovic** [comment]
![Screen Shot 2020-02-05 at 2.42.15 PM.png](https://images.zenhubusercontent.com/5ba135cf945aef49576d50f2/c2c04fe4-b380-4701-9cad-1370f735ef54)

Example of two up mode: https://bluemountain.princeton.edu/exist/apps/bluemountain/issue.html?titleURN=bmtnaae&issueURN=bmtnaae_1918-12_01#?#viewer&c=0&m=0&s=0&cv=0

**Comment by nikdragovic** [comment]
Page number data currently displaying in UV may help identify and properly display recto/verso.
<img width="1134" alt="Screen Shot 2020-02-06 at 5 24 58 AM" src="https://user-images.githubusercontent.com/12521534/73928474-24a5c680-48a1-11ea-98e0-da5431eb8e46.png">

**Comment by maxkadel** [comment]
A working example of two up mode http://nlw.azurewebsites.net/?manifest=http%3A%2F%2Fwellcomelibrary.org%2Fiiif%2Fb19684915%2Fmanifest&locale=en-GB#?c=0&m=0&s=0&cv=1&xywh=-2%2C-1737%2C11931%2C10599 

![image](https://user-images.githubusercontent.com/45948126/74037600-927aec80-498c-11ea-922a-72cc518a0a93.png)

**Comment by nikdragovic** [comment]
Thanks @maxkadel , this looks like what we want.

**Comment by maxkadel** [comment]
With a properly formed manifest, the one or two page view choice shows in UV

Properly formed manifest requires 
```
"@type":"sc:Sequence",
"viewingHint": "paged",
```

![image](https://user-images.githubusercontent.com/45948126/74265120-ef99d980-4ccf-11ea-8458-603925a9e144.png)

![image](https://user-images.githubusercontent.com/45948126/74265145-fd4f5f00-4ccf-11ea-9279-6ba1d7684289.png)

**Comment by bess** [comment]
This is done, IF the object from curate has a manifest that will support this. 

Today (11 Feb) Max and Bess had a phone call with @devanshu-m about how to get the right settings added to the curate manifest. He said that the curate project is not making any changes to the content of iiif manifests until some decisions are made about how manifests will ultimately be generated. He is waiting on feedback from @rotated8 and Emily. 

Should we call this done for now? Or should we put it on hold until we can get the curate manifests done?

**Comment by eporter23** [comment]
@nikdragovic @maxkadel @rotated8 @devanshu-m following up on a couple of conversations, if we now have UV support for this feature in Lux, and the dependency is on additional work from the Curate side, I think we can close this and revisit it later. We don't have a clear-cut way to manage some of this in Curate presently and want to focus on the higher priority manifest related work first.
